app/controllers/stores_controller.rb

```diff
@@ -49,8 +49,10 @@ def search
 
   def favorites
```

Comment:
Can this get moved to the store model and tested to make sure its working? 

From Rails Docs: The controller is responsible for making sense of the request and producing the appropriate output.